src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -287,8 +287,14 @@ protected function describeContainerEnvVars(array $envs, array $options = [])
     protected function describeEventDispatcherListeners(EventDispatcherInterface $eventDispatcher, array $options = [])
     {
         $event = \array_key_exists('event', $options) ? $options['event'] : null;
+        $dispatcherServiceName = \array_key_exists('dispatcher_service_name', $options) ? $options['dispatcher_service_name'] : null;
```

Comment:
```suggestion
        $dispatcherServiceName = $options['dispatcher_service_name'] ?? null;
```

Reply:
While I'm at it, should I do the same with the line above it? So they both look the same?